ui_testing/elements.py

```diff
@@ -231,19 +231,37 @@
                  'value': 'Next'},
         'test_units': {'method': 'id',
                        'value': 'selectedTestUnitsfield'},
-        'add': {'method': 'class_name',
-                'value': 'btn-primary',
-                'order': 0},
-        'save': {'method': 'class_name',
-                 'value': 'btn-primary',
-                 'order': 1},
+        # 'add': {'method': 'class_name',
+        #         'value': 'btn-primary',
+        #         'order': 0},
```

Comment:
why?

Reply:
The save button had those attributes: 
`'save': {'method': 'class_name',
                 'value': 'btn-primary',
                 'order': 1},`
If we kept on using the class_name, the order for the save button was going to be 0 [same for the add], so I didn't find any reference for the 'add' in the testplan page, I then changed the save button to be using 'id' instead of 'class_name'.